pkg/controller/gc.go

```diff
@@ -647,6 +647,10 @@ func (c *Controller) gcStaticRoute() error {
 	for _, route := range routes {
 		if route.CIDR != "0.0.0.0/0" && route.CIDR != "::/0" && c.ipam.ContainAddress(route.CIDR) {
 			klog.Infof("gc static route %s %s %s", route.Policy, route.CIDR, route.NextHop)
+			exist, err := c.ovnLegacyClient.NatRuleExists(route.CIDR)
```

Comment:
Just find some problem after merge. The logical should come before the `klog.Infof("gc static route %s %s %s", route.Policy, route.CIDR, route.NextHop)` and the err should be logged

Reply:
Sorry, It's my carelessness. I will create a new pr to repair it.